src/css/components/navigation.less

```diff
@@ -26,6 +26,7 @@
 .navbar-inverse .navbar-nav > .active > a:focus {
   color: @black-haze;
   background-color: transparent;
+  cursor: pointer;
```

Comment:
This is for active links – I kinda like the Bootstrap default of not showing a pointer for the currently active page.